<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="cad_username">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
padding-bottom: 7px;
</string>
    </property>
    <property name="placeholderText">
     <string>Nome do Funcionário</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cad_func_user">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
padding-bottom: 7px;
</string>
    </property>
    <property name="placeholderText">
     <string>Cargo do Funcionário</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cad_mat_user">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
padding-bottom: 7px;
</string>
    </property>
    <property name="placeholderText">
     <string>Matrícula do Funcionário</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>391</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-bottom-right-radius: 50px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="LogIn">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>EFETUAR CADASTRO</string>
     </property>
    </widget>
    <widget class="QLabel" name="cad_return">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>440</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cadFunc">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>510</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButtom#cadFunc{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 230, 219), stop:1 rgba(85, 81, 84, 226));
	color:rgba(255,255,255,210);
	border-radius: 5px; 

}

QPushButtom#cadFunc:hover{
}
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(77, 184, 204, 255), stop:1 rgba(255, 255, 255, 255));

QPushButtom#cadFunc:pressed{
	padding-left:5px;	
	padding-top:5px;
	background-color:rgba(150, 123, 111, 255);
}</string>
     </property>
     <property name="text">
      <string>CADASTRAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cadReset">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>510</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButtom#cadFunc{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 230, 219), stop:1 rgba(85, 81, 84, 226));
	color:rgba(255,255,255,210);
	border-radius: 5px; 

}

QPushButtom#cadFunc:hover{
}
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(77, 184, 204, 255), stop:1 rgba(255, 255, 255, 255));

QPushButtom#cadFunc:pressed{
	padding-left:5px;	
	padding-top:5px;
	background-color:rgba(150, 123, 111, 255);
}</string>
     </property>
     <property name="text">
      <string>RESET</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="cad_pass1_log">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
padding-bottom: 7px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Digite sua Senha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cad_pass2_log">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
padding-bottom: 7px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Repita sua Senha</string>
    </property>
   </widget>
   <widget class="QLabel" name="sahde_img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,50);
border-top-left-radius: 50px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_login">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img_log/girls_log.jpg);
border-top-left-radius: 50px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>girlslogin.png</pixmap>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>img_login</zorder>
   <zorder>cad_username</zorder>
   <zorder>cad_func_user</zorder>
   <zorder>cad_mat_user</zorder>
   <zorder>cad_pass1_log</zorder>
   <zorder>cad_pass2_log</zorder>
   <zorder>sahde_img</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cad_username</tabstop>
  <tabstop>cad_func_user</tabstop>
  <tabstop>cad_mat_user</tabstop>
  <tabstop>cad_pass1_log</tabstop>
  <tabstop>cad_pass2_log</tabstop>
  <tabstop>cadFunc</tabstop>
  <tabstop>cadReset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
